src/luaotfload-harf-plug.lua

```diff
@@ -993,10 +996,11 @@ local function set_tounicode()
 end
 
 -- FIXME: Move this into generic parts of luaotfload
+local utfchar = utf8.char
 local function get_glyph_info(n)
-  local props = properties[todirect(n)]
-  if not props then return end
-  return props and props.glyph_info or nil
+  n = todirect(n)
+  local props = properties[n]
+  return props and props.glyph_info or utfchar(getchar(n))
```

Comment:
I think returning `nil` here was fine, it will cause the engine to do the equivalent of `utfchar(getchar(n))` which might be slightly faster.

Reply:
No, if the callback exists but returns nil the engine uses `"` followed by the hexadecimal representation.

Reply:
I see.